src/monaco-languages.ts

```diff
@@ -492,4 +526,16 @@ export class MonacoLanguages implements Languages {
         return selector === model.languageId;
     }
 
+    protected matchLanguage(selector: string | DocumentFilter | DocumentSelector, languageId: string): boolean {
+        if (Array.isArray(selector)) {
+            return selector.findIndex(filter => this.matchLanguage(filter, languageId)) !== -1;
```

Comment:
fyi: `selector.some(...)` === `selector.findIndex(...) !== -1`

Reply:
Oh I would have used "some", but I saw findIndex used in matchModel and thought it's a coding standard you want to stick to. Will keep in mind for future PRs

Reply:
true, I will change it in: https://github.com/TypeFox/monaco-languageclient/pull/100